packages/accounts-base/accounts_server.js

```diff
@@ -1171,7 +1202,8 @@ Ap.config = function (options) {
 function setExpireTokensInterval(accounts) {
   accounts.expireTokenInterval = Meteor.setInterval(function () {
     accounts._expireTokens();
-    accounts._expirePasswordResetTokens();
+    accounts._expirePasswordResetTokens('reset');
+    accounts._expirePasswordResetTokens('enroll');
```

Comment:
Looks like this didn't get updated; this would perform the wrong action.  

`accounts._expirePasswordEnrollTokens();`

Right?